src/RuntimeBaseCommand.js

```diff
@@ -64,7 +67,19 @@ class RuntimeBaseCommand extends Command {
 
   async wsk (options) {
     if (!options) {
+      const authHandler = {
+        getAuthHeader: async () => {
+          await context.setCli({ 'cli.bare-output': true }, false) // set this globally
+          const env = getCliEnv()
+          console.debug(`Retrieving CLI Token using env=${env}`)
+          const accessToken = await getToken(CLI)
+
+          return `Bearer ${accessToken}`
+        }
+      }
       options = await this.getOptions()
+      options.auth_handler = authHandler
+      options.apihost = options.apihost ?? PropertyDefault.DEPLOYSERVICEURL
```

Comment:
Please add a comment in the issue description that this PR:
- do not merge
- this property needs to be updated with the final production url